<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alvaro Cubiella</author>
 <class>Frm_Inicio</class>
 <widget class="QWidget" name="Frm_Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>593</width>
    <height>262</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inicio</string>
  </property>
  <widget class="QLabel" name="lbl_Titulo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>571</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>571</width>
     <height>36</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Cargar estructura de campa単a</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="box_Seleccion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>191</width>
     <height>91</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>191</width>
     <height>91</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Seleccion</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>152</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="rbtn_Cargar">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cargar campa単a</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="rbtn_Nueva">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nueva campa単a</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box_Recientes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>571</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>571</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>571</width>
     <height>71</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Recientes:</string>
   </property>
   <widget class="QComboBox" name="cbox_Recientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>551</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>551</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>C:\\</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box_Camps">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>371</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>371</width>
     <height>71</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Seleccione campa単a</string>
   </property>
   <widget class="QComboBox" name="cbox_Cargar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>351</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>351</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>312</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_Cargar">
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Cancel">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="tree_json">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>256</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="animated">
    <bool>true</bool>
   </property>
   <property name="headerHidden">
    <bool>false</bool>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cbox_view">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>225</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ver archivo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
